docs/DeveloperGuide.md

```diff
@@ -132,11 +155,36 @@ The `Model`:
 
 ![Structure of the Storage Component](images/StorageClassDiagram.png)
 
-**API** : [`Storage.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/storagePerson/Storage.java)
+<div markdown="span" class="alert alert-info"> 
+**:information_source: Note:** ABC refers to `Deliverable`, `Meeting` and `Person`.
+
+</div>
+
+**API** : 
+[`StorageDeliverable.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/storage/deliverable/StorageDeliverable.java)
+[`StorageMeeting.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/storage/meeting/StorageMeeting.java)
+[`StoragePerson.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/storage/person/StoragePerson.java)
 
 The `Storage` component:
 * can save `UserPref` objects in json format and read it back.
-* can save the address book data in json format and read it back.
+* can save the data in json format and read it back.
+
+For saving files, storage follows this sequence:
+
+1. When `ABCBook` is updated, `StorageABCManagers` saves the newly updated book.
+1. The newly updated book is passed to `JsonSerliazableABCBook`.
+1. Each item in `ABCBook` is serialized by `JsonAdaptedABC` before overwriting the current Jsonfile
```

Comment:
XYZ
Is JsonFile a class/code? If yes need `` formatting. If not, maybe change to "current json file"

Reply:
All mentions of XYZ is a class/code